packages/components/src/components/ScalarMenu/ScalarMenu.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import { ScalarIcon, ScalarPopover } from '../../'
+import ScalarMenuProducts from './ScalarMenuProducts.vue'
+import ScalarMenuResources from './ScalarMenuResources.vue'
+</script>
+<template>
+  <ScalarPopover :placement="'bottom-start'">
```

Comment:
```suggestion
  <ScalarPopover placement="bottom-start">
```
static prop, probably has slightly better perf

Reply:
Oh I should make this overridable actually 😅